scripts/run-integration-tests.sh

```diff
@@ -3,7 +3,13 @@ set -e
 
 readonly PROJECT_ROOT=`cd $(dirname $0)/..; pwd`
 readonly IMAGE_NAME=pulsarctl-test
-readonly PULSAR_DEFAULT_VERSION="2.8.0-rc-202106151929"
+PULSAR_DEFAULT_VERSION="2.8.0-rc-202106151929"
+
+if [[ $PULSAR_MATRIX_VERSION  ]] && [ "$PULSAR_MATRIX_VERSION" != "default" ];then
```

Comment:
I think we just export the `PULSAR_VERSION` to the environment is enough? Because we check the pulsar version in line 13?

Reply:
I notice that the streamnative-ci will update the `PULSAR_DEFAULT_VERSION` value when a new version has been released, so I keep this compatibility.